ray-operator/config/samples/ray-job.interactive-mode.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: ray.io/v1
+kind: RayJob
+metadata:
+  name: rayjob-interactive-mode
+  namespace: default
+spec:
+  # InteractiveMode means KubeRay doesn't submit job for you.
```

Comment:
It's not clear enough. 

"InteractiveMode means KubeRay doesn't submit job for you." -> what's the user's responsibility and what KubeRay will do for users in this mode?

Reply:
done